11ty/definitions/dumb.md

```diff
@@ -5,6 +5,7 @@ flag:
   text: 'ableist language'
   level: avoid
 defined: true
+excerpt: non-verbal, non-speaking, silent, refraining from speaking or unable to speak; sometimes __mute__ which can also be ableist; colloquially used to mean unintelligent or senseless
```

Comment:
```suggestion
excerpt: avoid. ableist. colloquially used to mean unintelligent or senseless
```

Reply:
We add avoid in the template already, I think by adding it to the excerpt it will show up twice.